editor/src/components/canvas/canvas-strategies/strategies/flow-reorder-helpers.ts

```diff
@@ -34,51 +35,97 @@ export function areAllSiblingsInOneDimensionFlexOrFlow(
   target: ElementPath,
   metadata: ElementInstanceMetadataMap,
 ): boolean {
-  const targetElement = MetadataUtils.findElementByElementPath(metadata, target)
   const siblings = MetadataUtils.getSiblings(metadata, target) // including target
-  if (targetElement == null || siblings.length === 1) {
+  if (siblings.length === 1) {
     return false
   }
 
-  if (MetadataUtils.isParentYogaLayoutedContainerForElement(targetElement)) {
-    const flexDirection = targetElement.specialSizeMeasurements.parentFlexDirection
-    const targetDirection =
-      flexDirection === 'row' || flexDirection === 'row-reverse' ? 'horizontal' : 'vertical'
-    const shouldReverse = flexDirection?.includes('reverse') ?? false
-    const frames = mapDropNulls((sibling) => {
+  return staticContainerDirections(EP.parentPath(target), metadata) !== 'non-1d-static'
+}
+
+export function staticContainerDirections(
+  container: ElementPath,
+  metadata: ElementInstanceMetadataMap,
+):
+  | {
```

Comment:
Could we separate out this into a type somewhere?

Reply:
we deliberately did not want to make this an explicit type, because we wanted to avoid looking like this is part of an API or a sensible collection of values. it's mostly necessity-driven that these data must come from one function call

Reply:
https://github.com/concrete-utopia/utopia/pull/2738#discussion_r1018958369

Reply:
fixed!